integration/v3_auth_test.go

```diff
@@ -270,3 +270,29 @@ func authSetupRoot(t *testing.T, auth pb.AuthClient) {
 		t.Fatal(err)
 	}
 }
+
+func TestV3AuthNonAuthorizedRPCs(t *testing.T) {
+	defer testutil.AfterTest(t)
+	clus := NewClusterV3(t, &ClusterConfig{Size: 1})
+	defer clus.Terminate(t)
+
+	nonAuthed, err := clientv3.New(clientv3.Config{Endpoints: clus.Client(0).Endpoints()})
```

Comment:
can this just use `clus.Client(0).KV`? server integration tests should try to use the raw grpc interface..